OPHD/Population/Population.cpp

```diff
@@ -98,156 +99,146 @@ int Population::adults() const
 void Population::spawn_children(int morale, int residences, int nurseries)
 {
 	if (residences < 1 && nurseries < 1) { return; }
+	if (mPopulation[PersonRole::ROLE_SCIENTIST] + mPopulation[PersonRole::ROLE_WORKER] < 1) { return; }
```

Comment:
I would probably use `<= 0`, rather than `< 1`, as it kind of gets more to the heart of the matter. This applies multiple times to the guard clauses of all population functions.